containers/src/Kokkos_DualView.hpp

```diff
@@ -14,7 +14,13 @@
 #define KOKKOS_IMPL_PUBLIC_INCLUDE_NOTDEFINED_DUALVIEW
 #endif
 
+#include <Kokkos_Macros.hpp>
+#ifdef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULES
+import kokkos.core;
+import kokkos.core_impl;
+#else
 #include <Kokkos_Core.hpp>
+#endif
 #include <impl/Kokkos_Error.hpp>
 
```

Comment:
`uintptr_t` is used in this file
```suggestion
#include <impl/Kokkos_Error.hpp>

#include <cstdint>

```